scipy/sparse/csgraph/tests/test_shortest_path.py

```diff
@@ -33,9 +33,9 @@
 directed_2SP_0_to_3 = [[-9999, 0, -9999, 1, -9999],
                        [-9999, 0, -9999, 4, 1]]
 
-directed_sparse_zero_G = scipy.sparse.csr_matrix(([0, 1, 2, 3, 1], 
-                                            ([0, 1, 2, 3, 4], 
-                                             [1, 2, 0, 4, 3])), 
+directed_sparse_zero_G = scipy.sparse.csr_matrix(([0, 1, 2, 3, 1],
+                                            ([0, 1, 2, 3, 4],
+                                             [1, 2, 0, 4, 3])),
```

Comment:
optionally, we could fix the indentation here, while the lines are already being touched.